### DISCUSSION TITLE
Is it possible to use capture with templater at the same time?

### DISCUSSION THREAD

**Discussion by cmdrSpectral** [chhoumann/quickadd#389] [Q&A]
Hi all,
Kind a obsidian newbie here, but there is an issue that bugs me for two days. I would like to achieve one single trick -> append a line (or a couple of lines) based on a simple template. Open the file, and have the cursor properly placed. (I use it as micro journal -> single file fil dates, not a part of a daily note)
It seems impossible at the moment. The closes solution I got was to use "Capture" without using {{value}}. I got many lovely options on data placement, like the end of file, front of file, section of file, create section as needed and so on. Works like a charm - I just can't place my cursor. (I've seen the topic and templater solution didn't work)
I can you templater options but as they work great with "Template" type, the inline templater stuff (like <% tp.file.cursor(1) %>) get omitted. 
My question is: can I have a "template" add with all the options "capture" has? Or can I have templater code evaluated when using capture format?

Additional small request -> can there be an option not to open a copy o fthe file if the file is already open and just focus on that tab? 
I add data to one existing file, and sometimes it's open, and sometimes it's not. If it's already open, I do get a second tab with the file, focus switch would be enough.

Thanks

**Comment by chhoumann** [comment]
Hi @jgrobelny-dev 

At this point, I'm strongly considering making a feature like Templater's cursor feature.
But for now, you can use a macro to activate template a short duration after your capture.

As for your request:
That is something I'm looking to fix.

**Comment by chhoumann** [comment]
Coming back to this very late - sorry about that. Both things you asked for are native QuickAdd behavior now:

1. Cursor placement: since v2.14.0, when a Capture opens the target file in an editable mode, QuickAdd places the cursor at the end of the inserted text - so you can keep typing right where the capture landed. No Templater cursor markers needed.
2. Duplicate tabs: if the file is already open, QuickAdd focuses that existing tab instead of opening a second one (fixed in v1.12.0). And when it isn't open yet, the _File Opening Location_ dropdown lets you pick where it goes, including _Reuse current tab_.

Docs: https://quickadd.obsidian.guide/docs/Choices/CaptureChoice#opening-the-captured-file

Closing as resolved.
